homeassistant/components/switchbot_cloud/__init__.py

```diff
@@ -142,6 +142,31 @@ async def make_device_data(
         )
         devices_data.sensors.append((device, coordinator))
 
+    if isinstance(device, Device) and device.device_type in [
+        "Relay Switch 2PM",
+    ]:
+        coordinator = await coordinator_for_device(
+            hass, entry, api, device, coordinators_by_id
+        )
+
+        device_switch_1: Device = Device(
+            deviceId=f"{device.device_id}-1",
+            deviceName=f"{device.device_name}-1",
+            deviceType=device.device_type,
+            hubDeviceId=device.hub_device_id,
+        )
+        device_switch_2: Device = Device(
+            deviceId=f"{device.device_id}-2",
+            deviceName=f"{device.device_name}-2",
+            deviceType=device.device_type,
+            hubDeviceId=device.hub_device_id,
+        )
```

Comment:
Why do we create library objects in the integration?

Reply:
Relay Switch 2PM doesn't not like normal switch entity, it has two switches inside, they should not controlled by on button,  I have to split it into two switch entity